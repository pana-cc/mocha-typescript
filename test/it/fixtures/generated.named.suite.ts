import { suite, test } from "../../../index";

class TestSuite {

  @test myTest() {}
}

suite("Named Suite")(TestSuite);
