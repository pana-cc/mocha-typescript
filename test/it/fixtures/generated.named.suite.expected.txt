
  Named Suite
    âœ“ myTest

  1 passing
